test_root/.jenkins-coveragerc

```diff
@@ -1,4 +1,8 @@
 [paths]
-source =
+build_jenkins_source =
```

Comment:
Please add a comment in this file explaining that these are the names of the jenkins servers, and the jobs on each. And that the groovy script on each of the jobs will reference the corresponding section in this file.

At least that's what I think is happening here, I could be wrong...